lua/apisix/plugins/kafka-logger.lua

```diff
@@ -56,9 +55,12 @@ local function log(premature, conf, log_message)
         return
     end
 
-    local broker_list = {
-        { host = conf.broker_host, port = conf.port }
-    }
+    if #conf.broker_list == 0 then
```

Comment:
the `conf. broker_list` is a hash object, we need to use `table.nkeys` to fetch the number of attributes.

`#` is useful for array tables.